spec/controllers/idv/session_errors_controller_spec.rb

```diff
@@ -160,7 +160,7 @@
       let(:user) { create(:user) }
 
       before do
-        RateLimiter.new(rate_limit_type: :proof_address, user: user).increment!
+        RateLimiter.new(rate_limit_type: :idv_resolution, user: user).increment!
```

Comment:
I think this was accidentally changed in https://github.com/18F/identity-idp/pull/5546

Reply:
WDYT about moving fixing this into a separate PR?

Reply:
sure!

Reply:
opened https://github.com/18F/identity-idp/pull/9222